modules/29_CES_parameters/calibrate/preloop.gms

```diff
@@ -716,6 +716,103 @@ pm_cesdata(t,regi_dyn29,"kap","eff")
 
 display "after change cap eff consistency", pm_cesdata, pm_cesdata_putty;
 
+*** If the value (quantity x price) of either en or kap, or the sum of both,
+*** exceed the quantity of inco (all of which would result in negative labour
+*** prices), scale these prices down accordingly, and warn this is happening.
```

Comment:
I assume, you tested it. I am always a bit reluctant scaling manually some values. I am afraid that we will have to adjust it too often. Does the auto-scaling not do its job?

Reply:
Yes, I tested it.  This is no manual scaling, the scaling factors are calculated endogenously, there are no values to adjust.  There was no automatic scaling prior to this patch.